docs/resources/downtime.md

```diff
@@ -50,6 +50,7 @@ resource "datadog_downtime" "foo" {
 
 ### Optional
 
+- **active_child_id** (Number) The id corresponding to the downtime object definition of the active child for the original parent recurring downtime. This field will only exist on recurring downtimes.
```

Comment:
if we include this, I think it should be actually read only.

Reply:
Bumping on this, is this part auto generated or a copy/paste?

Reply:
The doc is auto generated. But to avoid this, we should remove the line below since the attribute is read only: https://github.com/DataDog/terraform-provider-datadog/blob/f06954f01639447aaeab733c6488555282a96f34/datadog/resource_datadog_downtime.go#L253